install.sh

```diff
@@ -1192,29 +1171,29 @@ install_hooks_into_repo() {
     else
         # Getting the working tree (no external .git directories)
         # see https://stackoverflow.com/a/38852055/293195
-        TARGET_ROOT=$(git -C "${TARGET}" rev-parse --show-toplevel 2>/dev/null)
+        TARGET_ROOT=$(git -C "$TARGET" rev-parse --show-toplevel 2>/dev/null)
```

Comment:
Removed some uneeded braces....

Reply:
/nit: I usually err on the side of having unnecessary braces rather than omitting braces, to avoid problems with missing braces on a refactor. I'm not too fussed about this one in particular here, so I'm fine either way, just wanted to tell you for future changes. :)

Reply:
Ok, but with that in mind, shouldn't the whole code base use  `${...}` in favor of `$...`?  You are right, during regex/replace we ofther forget the `${. *}` case, hm...

Reply:
Yeah, that might be a good change for me, but perhaps it's more personal preference than anything else. Right now I think we have this mixed here and there, so I'm not too bothered.
Maybe if I find some time I'll change all of them and add a test to enforce it. 🤷‍♂️